### ISSUE TITLE
[Transporter Error Output]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)

### ISSUE THREAD

**Issue by janpio** [fastlane/fastlane#13306]
This is a follow up to https://github.com/fastlane/fastlane/issues/13305, where the error mentioned in the title and above appeared multiple times:

```
[15:32:54]: [Transporter Error Output]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)
[15:32:54]: Transporter transfer failed.
[15:32:54]:
[15:32:54]: Sign in with the app-specific password you generated. If you forgot the app-specific password or need to create a new one, go to appleid.apple.com (-22938)
```
(go to https://github.com/fastlane/fastlane/issues/13305 for stacktraces etc as well)

@jhoon in https://github.com/fastlane/fastlane/issues/13281#issuecomment-418806266 you reported this problem. 
@zolotyx in https://github.com/fastlane/fastlane/issues/13284#issuecomment-419014887 you reported this problem as well. 

How did you set the password to get this error? 
And this worked before upgrading to fastlane 1.203.x?

**Comment by m17603048828** [comment]
hello  solve it 
ENV["FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD"] = "xxxxxx"

**Comment by pedro380085** [comment]
Same error here.

**Comment by janpio** [comment]
Then the same questions apply to you @pedro380085: 
How did you set the password to get this error?
And this worked before upgrading to fastlane 1.203.x?

In the other issue you posted at https://github.com/fastlane/fastlane/issues/13305#issuecomment-419193418:
> Previously it added a shell so we could create a new 2FA session. Now it just fails.
What does "added a shell" mean?

**Comment by pedro380085** [comment]
hi @janpio.

To solve this error I've set `ENV["FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD"] = "xxxxxx"`.


**And this worked before upgrading to fastlane 1.203.x?**
Yes, before 1.203.x it prompted me with a new 2FA process. I had a shell I would type the 6 digits generated to me. By added a shell, I mean a prompt a could type the 6 digits. Sorry It was not clear on the other thread.

**Comment by janpio** [comment]
Ok, I might have understood what is going on here. If you want to follow along, read here: https://github.com/fastlane/fastlane/issues/13305#issuecomment-419466779

**Comment by joshdholtz** [comment]
This _might_ be fixed now in `2.104.0` 😊 🤞
